app/src/main/java/com/hammer/talkbbokki/presentation/navigation/TalkbbokkiNavHost.kt

```diff
@@ -57,8 +57,8 @@ fun TalkbbokkiNavHost(
             navigateToSuggestion = { navController.navigate(SuggestionDestination.route) }
         )
         topicListGraph(
-            navigateToDetail = { level, id, topic ->
-                navController.navigate(DetailDestination.route + "?level=${level}&id=${id}&topic=${topic}")
+            navigateToDetail = { level, id, tag, topic, shareLink ->
+                navController.navigate(DetailDestination.route + "?level=${level}&id=${id}&tag=${tag}&topic=${topic}&shareLink=${shareLink}")
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)